README.md

```diff
@@ -770,6 +711,8 @@ You can define your own custom toolsets to extend the functionality of your setu
 # Example: ["path/to/your/custom_toolset.yaml"]
 #custom_toolsets: ["examples/custom_toolset.yaml"]
 ```
+
+There is also detailed documentation in [Robusta's online docs](https://docs.robusta.dev/master/configuration/holmesgpt/custom_toolsets.html).
```

Comment:
This link is broken

Reply:
Is that just because the docs changes are not merged to master yet?

Reply:
Yes. To test the future links you can replace `master` with `improve_holmes_docs`